channeldb/channel_test.go

```diff
@@ -157,6 +158,25 @@ func remoteHtlcsOption(htlcs []HTLC) testChannelOption {
 	}
 }
 
+// loadFwdPkgs is a helper method that reads all forwarding packages for a
+// particular packager.
+func loadFwdPkgs(t *testing.T, db kvdb.Backend,
+	packager FwdPackager) []*FwdPkg {
+
+	var (
+		fwdPkgs []*FwdPkg
+		err     error
+	)
+
+	err = kvdb.View(db, func(tx kvdb.RTx) error {
```

Comment:
Has been a while since this PR was touched last, but all the calls to `View/Update` will need to be updated w/ the new extra arg to handle transaction retries if `etcd` is being used.

Reply:
Fixed!